src/api-reference/request/v4.examples.markdown

```diff
@@ -148,20 +149,103 @@ Responde Body:
     "currency": "USD"
   },
   "travelAgency": {
-    "href": "https://rqa3-raas.concurasp.com/v4/travelagencies/2EC038D7C3CBBE4ABA0914425064D34F",
+    "href": "https://us.api.concursolutions.com/v4/travelagencies/2EC038D7C3CBBE4ABA0914425064D34F",
     "id": "2EC038D7C3CBBE4ABA0914425064D34F",
-    "template": "https://rqa3-raas.concurasp.com/v4/travelagencies/{id}"
+    "template": "https://us.api.concursolutions.com/v4/travelagencies/{id}"
   },
   "operations": [
     {
       "rel": "submit",
-      "href": "https://rqa3-raas.concurasp.com/v4/requests/CED5E9CD8FC1424488F9331ACF956E73/submit{?companyId}"
+      "href": "https://us.api.concursolutions.com/v4/requests/CED5E9CD8FC1424488F9331ACF956E73/submit{?companyId}"
     }
   ]
 }
 ```
 [Back to top](#request-examples)
 
+#### <a name="request-resource-get-list"></a>Get the list of existing Requests
+#### GET /v4/requests
+```
+curl -X GET https://us.api.concursolutions.com/travelrequest/v4/requests?view=ALL&limit=10&start=0 --header 'Accept: application/json' --header 'Authorization: Bearer {token}'
```

Comment:
We need to remove references to specific tools. Follow the example given in the [Documentation Template](https://github.com/concur/developer.concur.com/blob/preview/src/document-template.markdown).

Reply:
Is this really required ? curl is a pretty standard cross platform tool. I think it really helps developers to figure out what a valid call is.

Reply:
Hi Charlie. There seems to be some current discussion wth Carol on this subject, however as we are externalizing the Get list of Request this saturday, and knowing that removing the curls will require dev time not planned and possible before saturday - of confirmed, please display the documentation with Curl, and for the october release (there will be new documentation update) we'll see what is really necessary and what can be done on the dev side to make it possible.
Thanks in advance.